recipe_scrapers/usapears.py

```diff
@@ -25,6 +28,30 @@ def ingredients(self):
             for paragraph in ingredient_elements
         ]
 
+    def nutrients(self):
+        container = self.soup.find("ul", {"itemprop": "nutrition"})
+        if not container:
+            raise ElementNotFoundInHtml("Could not find nutritional info container")
+
+        results = {}
+        redundant_pattern = r"<strong>(.+)[:] </strong>"
+        for item in container.find_all("li", {"itemprop": True}):
+            nutrient = item["itemprop"]
+            content = "".join(str(elem) for elem in item.children)
+            if re.match(redundant_pattern, content):
+                content = re.sub(redundant_pattern, "", content)
+            results[nutrient] = content
+
+        corrections = {
+            "carbohydrates": "carbohydrateContent",
+            "protein": "proteinContent",
+        }
```

Comment:
```suggestion
            "fat": "fatContent",
        }
```

Looked through some other recipes on the page [at least one](https://usapears.org/recipe/crispy-potato-latkes-with-spiced-pear-compote/) lists `fatContent` as `fat`